### PR TITLE
Add mising keepdim with fixed typo

### PR DESCRIPTION
**Pull request by julien-blanchon**
#### Changes

Add missing `keepdim` argument in some augmentation constructors to enable keepdim.

Fixes # (issue)

#### Type of change

- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [x] 🔬 New feature (non-breaking change which adds functionality)

#### Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by julien-blanchon** [comment]
https://github.com/kornia/kornia/compare/master...julien-blanchon:patch-5

**Comment by julien-blanchon** [comment]
https://github.com/kornia/kornia/pull/1488

Sorry for the 3 PR ^^

File: kornia/augmentation/augmentation.py [hunk 1]
```diff
@@ -1988,8 +2037,9 @@ def __init__(
         return_transform: bool = False,
         same_on_batch: bool = False,
         p: float = 0.5,
+        keepdim: bool = False
     ) -> None:
-        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0)
+        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0, keepdim=False)
```
Comment (edgarriba) [inline kornia/augmentation/augmentation.py:2042]: Format this one too

**Review by edgarriba — COMMENTED** [review]
Missing one format and all good

File: kornia/augmentation/augmentation.py (hunk 1 above)
Comment (shijianjian) [inline kornia/augmentation/augmentation.py:2042]: Why always False?

Comment (julien-blanchon) [reply kornia/augmentation/augmentation.py:2042]: Oups it's a typo, should be `keepdim=keepdim`, I will re-check everything tomorrow if you want.

**Pull request closed without merging**